README.md

```diff
@@ -94,12 +94,8 @@ apply plugin: 'kover'
 ```
 </details>
 
-### Apply plugin to multi-module project
-To apply the plugin to all modules in the project, you need to apply the plugin only to the root module, as shown [above](#apply-plugin-to-single-module-project).
-
-**There are no dependencies between tasks from different modules, they are executed independently.**
-
-**Cross-module tests are not supported in reports and validation yet. For each test, only the classpath belonging to the current module is taken.**
+### Apply plugin to multi-module build
+To apply the plugin to all modules, you just need to apply the plugin only to the root module, as shown [above](#apply-plugin-to-single-module-build).
```

Comment:
I'd add here something like 'Applying the plugin to submodules if you have already applied it to the root module will cause configuration errors'